### ISSUE TITLE
Lquery aggregate functions doesn't have proper types in luna

### ISSUE THREAD

**Issue by sylwiabr** [luna/Dataframes#84]
<img width="537" alt="zrzut ekranu 2018-10-17 o 12 26 22" src="https://user-images.githubusercontent.com/12892578/47080276-e0a03580-d207-11e8-9fe7-c96de1e60cd7.png">

**Comment by mwu-tow** [comment]
What is "proper" type? What behavior do you expect here?

**Comment by sylwiabr** [comment]
AggregateFunction type since nothing else will work

**Comment by mwu-tow** [comment]
I don't think this is something specific to the `rolling` function (or Dataframes in general). Nor something we are able to fix. Luna would need to allow explicit type annotations first.
